_i18n/en/resources/developer-guides/daemon-rpc.md

```diff
@@ -563,270 +783,1636 @@ $ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","me
-  },
-  ...]
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "status": "OK"
+  }
 }
 ```
 
 
-### **/stop_daemon**
+### **sync_info**
 
-Send a command to the daemon to safely disconnect and shut down.
+Get synchronisation informations
+
+Alias: *None*.
 
 Inputs: *None*.
 
 Outputs:
 
+* *height* - unsigned int;
+* *peers* - array of peer structure, defined as follows:
+  * *info* - structure of connection info, as defined in [get_connections](#get_connections)
+* *spans* - array of span structure, defined as follows (optional, absent if node is fully synced):
+  * *connection_id* - string; Id of connection
+  * *nblocks* - unsigned int; amount of blocks being synced
+  * *rate* - unsigned int; connection rate
+  * *remote_address* - string; peer address the node is spanning from
+  * *size* - unsigned int; synchronisation size
+  * *speed* - unsigned int; connection speed
+  * *start_block_height* - unsigned int; starting block height being synchronised
 * *status* - string; General RPC error code. "OK" means everything looks good.
+* *target_height* - unsigned int; target height the node is syncing to (optional, absent if node is fully synced)
 
 Example:
 
 ```
-$ curl -X POST http://127.0.0.1:18081/stop_daemon -H 'Content-Type: application/json'
+$ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"sync_info"}' -H 'Content-Type: application/json'
 
 {
-  "status": "OK"
-}
-```
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "height": 1563543,
+    "peers": [{
+      "info": {
+        "address": "70.109.53.128:60064",
+        "avg_download": 0,
+        "avg_upload": 5,
+        "connection_id": "204067223b9b3415c265dd25ad29ee48",
+        "current_download": 0,
+        "current_upload": 1,
+        "height": 1559975,
+        "host": "70.109.53.128",
+        "incoming": true,
+        "ip": "70.109.53.128",
+        "live_time": 38,
+        "local_ip": false,
+        "localhost": false,
+        "peer_id": "96b8545dbc7a8866",
+        "port": "60064",
+        "recv_count": 1580,
+        "recv_idle_time": 28,
+        "send_count": 203603,
+        "send_idle_time": 8,
+        "state": "state_normal",
+        "support_flags": 1
+      }
+    },{
+      "info": {
+        ...
+      }
+    },{
+      ...
+    },{
+      ...
+    },{
+      ...
+    }],
+    "status": "OK",
+    "target_height": 1564067
+  }
+}
+```
+
+
+### **get_txpool_backlog**
+
+Get all transaction pool backlog
+
+Alias: *None*.
+
+Inputs: *None*.
+
+Outputs:
+
+* *backlog*: array of structures *tx_backlog_entry* (in binary form):
```

Comment:
Hmm, I guess I did not realize it was binary. I should either rename it with ".bin" like the others, or make this json.

Reply:
I confirm the backlog is binary.
Sample:
```
{
  "id": "0",
  "jsonrpc": "2.0",
  "result": {
    "backlog": "<88>3^@^@^@^@
```